config/jobs/kubernetes/sig-cloud-provider/gcp/gce-conformance.yaml

```diff
@@ -1,4 +1,41 @@
 periodics:
+- interval: 3h
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-gce-conformance-latest
+  annotations:
+    fork-per-release: "true"
+    fork-per-release-replacements: "--extract=ci/latest-fast -> --extract=ci/latest-{{.Version}}"
+    testgrid-dashboards: conformance-all, conformance-gce
+    testgrid-tab-name: Conformance - GCE - master
+    description: Runs conformance tests using kubetest against kubernetes master on GCE
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  decorate: true
+  decoration_config:
+    timeout: 220m
+  spec:
+    containers:
+    - command:
+        - runner.sh
+        - /workspace/scenarios/kubernetes_e2e.py
+      args:
+      - --extract=ci/latest-fast
```

Comment:
are you sure this is the right parameter?

Reply:
I've just restored the values from when it was removed. https://github.com/kubernetes/test-infra/commit/742982210cd9bb6295801d422f012c4dc29e25ba

Reply:
I just bumped the image version